Firebase/android/app/src/main/res/layout/merge_welcome_view.xml

```diff
@@ -4,19 +4,49 @@
   android:layout_height="match_parent">
 
   <ImageView
+    android:id="@+id/userAvatar"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:minWidth="80dp"
-    android:minHeight="80dp"
-    android:layout_margin="16dp"
-    android:src="@mipmap/ic_launcher"/>
+    android:layout_marginTop="86dp"
+    android:minHeight="86dp"
+    android:minWidth="86dp"
+    android:scaleType="fitCenter" />
 
   <TextView
     android:id="@+id/welcomeMessageView"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:gravity="center"
+    android:textColor="@color/text_black"
+    android:textSize="16sp" />
+
+  <TextView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="24dp"
+    android:layout_marginLeft="16dp"
+    android:layout_marginRight="16dp"
+    android:layout_marginTop="48dp"
+    android:gravity="center"
+    android:text="@string/enjoy_the_awesomness"
+    android:textColor="@color/text_black"
+    android:textSize="16sp" />
+
+  <TextView
+    android:id="@+id/loginButton"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:background="@drawable/oval_shape_primary"
+    android:fontFamily="sans-serif-medium"
     android:gravity="center"
-    style="@style/TextAppearance.AppCompat.Headline"
-    android:layout_margin="16dp"/>
+    android:textColor="@android:color/white"
+    android:textSize="18sp"
+    android:text="@string/login"
```

Comment:
Humm good point it might have been a missunderstanding with @qqipp. Could you confirm with her ?